account_reconcile_oca/views/account_bank_statement_line.xml

```diff
@@ -213,12 +222,42 @@
                 <field name="foreign_currency_id" invisible="1" />
                 <field name="company_currency_id" invisible="1" />
                 <field name="currency_id" invisible="1" />
+                <field name="reconcile_mode" invisible="1" />
                 <field
                     name="reconcile_data_info"
                     nolabel="1"
                     widget="account_reconcile_oca_data"
                     class="w-100"
                 />
+                <field
+                    name="reconciled_line_ids"
+                    nolabel="1"
+                    attrs="{'invisible': [('reconcile_mode', '!=', 'keep')]}"
```

Comment:
```suggestion
                    attrs="{'invisible': ['|', ('reconcile_mode', '!=', 'keep'), ('reconciled_line_ids', '=', [])]}"
```

Reply:
@JordiMForgeFlow fixed!

Reply:
@JordiMForgeFlow Made a mistake. Now should be fixed.